DZ_3_1/templates/register.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Регистрация</title>
+</head>
+<body>
+    <h1>Регистрация</h1>
+    <form method="POST" action="/">
+        {{ form.hidden_tag() }}
+        <div>
+            {{ form.first_name.label }} {{ form.first_name() }}
+        </div>
+        <div>
+            {{ form.last_name.label }} {{ form.last_name() }}
+        </div>
+        <div>
+            {{ form.email.label }} {{ form.email() }}
+        </div>
+        <div>
+            {{ form.password.label }} {{ form.password() }}
+        </div>
+        <div>
+            {{ form.confirm_password.label }} {{ form.confirm_password() }}
+        </div>
+        <div>
+            {{ form.submit() }}
+        </div>
```

Comment:
Что если у вас будет десятки полей в форме? Шаблон должен быть масшабиремым
```suggestion
        {{ form.csrf_token() }}
        {% for field in form if field.name != 'csrf_token' %}
        <div>
            {{ field.label }} {{ field() }}
        </div>
        <ul class="alert alert-danger">
            {% for error in field.errors %}
            <li>{{ error }}</li>
            {% endfor %}
        </ul>
        {% endfor %}
```
Причем строки
``html
        <ul class="alert alert-danger">
            {% for error in field.errors %}
            <li>{{ error }}</li>
            {% endfor %}
        </ul>
```
как раз для того, чтобы выводить сообщения, если форма не прошла валидацию